aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move

```diff
@@ -0,0 +1,1048 @@
+        let nft_mint_config = borrow_global_mut<NFTMintConfig>(@post_mint_reveal_nft);
+        assert!(signer::address_of(admin) == nft_mint_config.admin, error::permission_denied(ENOT_AUTHORIZED));
+        nft_mint_config.treasury = new_treasury_address;
+    }
+
+    /// Set up and create the destination collection.
+    public entry fun set_collection_config_and_create_collection(
+        admin: &signer,
+        collection_name: String,
+        collection_description: String,
+        collection_maximum: u64,
+        collection_uri: String,
+        collection_mutate_config: vector<bool>,
+        token_name_base: String,
+        royalty_payee_address: address,
+        token_description: String,
+        token_maximum: u64,
+        token_mutate_config: vector<bool>,
+        royalty_points_den: u64,
+        royalty_points_num: u64,
+        public_mint_limit: Option<u64>,
+    ) acquires NFTMintConfig {
+        let nft_mint_config = borrow_global_mut<NFTMintConfig>(@post_mint_reveal_nft);
+        assert!(signer::address_of(admin) == nft_mint_config.admin, error::permission_denied(ENOT_AUTHORIZED));
+
+        assert!(vector::length(&collection_mutate_config) == 3 && vector::length(&token_mutate_config) == 5, error::invalid_argument(EVECTOR_LENGTH_UNMATCHED));
+        assert!(royalty_points_den > 0 && royalty_points_num < royalty_points_den, error::invalid_argument(EINVALID_ROYALTY_NUMERATOR_DENOMINATOR));
+        assert!(!exists<CollectionConfig>(@post_mint_reveal_nft), error::permission_denied(ECOLLECTION_ALREADY_CREATED));
+
+        let nft_mint_config = borrow_global_mut<NFTMintConfig>(@post_mint_reveal_nft);
+        let resource_signer = create_signer_with_capability(&nft_mint_config.signer_cap);
+
+        // Create the destination collection that holds the unique art NFT.
+        create_collection(&resource_signer, collection_name, collection_description, collection_uri, collection_maximum, collection_mutate_config);
+
+        move_to(&resource_signer, CollectionConfig {
+            collection_name,
+            collection_description,
+            collection_maximum,
+            collection_uri,
+            collection_mutate_config,
+            token_name_base,
+            royalty_payee_address,
+            token_description,
+            token_maximum,
+            token_mutate_config: create_token_mutability_config(&token_mutate_config),
+            royalty_points_den,
+            royalty_points_num,
+            tokens: big_vector::new<TokenAsset>(128),
+            public_mint_limit,
+        });
+
+        // Create the source certificate collection and token.
+        let source_collection_name = collection_name;
+        string::append_utf8(&mut source_collection_name, b" - source collection");
+        create_collection(&resource_signer, source_collection_name, collection_description, utf8(b""), collection_maximum, collection_mutate_config);
```

Comment:
collection_description is not empty? line 245 said it is empty string.

Reply:
line 245 is token description